setup.cfg

```
@@ -2,7 +2,7 @@
 universal = 1
 
 [metadata]
-description-file = README.md
+description_file = README.md
```

To be honest, now that I have re-read the documentation I can see that `description_file` is not even a valid field for setuptools.

This is the closest:

```suggestion
long_description = file: README.md
```

Docs: https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#configuring-setuptools-using-setup-cfg-files